### PR TITLE
feat(OO-bot): Improve settlement logs

### PR DESCRIPTION
**Pull request by chrismaree**
Settlement logs can be really messy. this was not an issue before as we were not settling a lot of request. now, however, we are settling requests on a number of chains. To improve quality of life, this PR refines how settlement logs are produced.

Sample message boy output mark-down:

[0x3C4...4293BC](https://etherscan.io/address/0x3C44CdDdB6a900fa2b585dd299e03d12FA4293BC) proposed a price of1.00 proposal for TEST8DECIMALS at timestamp 1643168842 with Ancillary data 0x which has been settled! proposer payout is 2.00 tx: [0xcad...cfc567](https://etherscan.io/tx/0xcad8a94db472de0aff860dd8db7b594f59cb0d522a0b8499051b2eaabfcfc567)

### CODE REVIEW

File: packages/monitors/src/OptimisticOracleContractMonitor.js [hunk 1]
```diff
@@ -141,7 +141,10 @@ class OptimisticOracleContractMonitor {
           this.oracleType === OptimisticOracleType.OptimisticOracle ? event.proposer : event.request.proposer,
           this.contractProps.networkId
         ) +
-        ` proposed a price for the request made by ${event.requester} at the timestamp ${event.timestamp} for the identifier: ${event.identifier}. ` +
```
Comment (chrismaree) [inline packages/monitors/src/OptimisticOracleContractMonitor.js:144]: changed this to be a mark-down link. consider the message below and how this change would have improved it
image

File: packages/optimistic-oracle/src/proposer.js [hunk 2]
```diff
@@ -435,30 +443,45 @@ class OptimisticOracleProposer {
...
-        ...receipt.events.Settle.returnValues,
-        identifier: this.hexToUtf8(receipt.events.Settle.returnValues.identifier),
-        ancillaryData: receipt.events.Settle.returnValues.ancillaryData || "0x",
-      };
-      this.logger.info({
-        at: "OptimisticOracleProposer#settleRequests",
-        message: "Settled proposal or dispute!⛑",
-        priceRequest,
-        payout:
+      const mrkdwn =
```
Comment (chrismaree) [inline packages/optimistic-oracle/src/proposer.js:451]: this is the main change of this PR.

File: packages/optimistic-oracle/test/proposer.js [hunk 3]
```diff
@@ -441,9 +441,9 @@ describe("OptimisticOracle: proposer.js", function () {
         await verifyState(OptimisticOracleRequestStatesEnum.SETTLED, identifiersToTest[i], ancillaryDataAddresses[i]);
       }
       assert.equal(lastSpyLogLevel(spy), "info");
-      assert.isTrue(spyLogIncludes(spy, -1, "Settled proposal or dispute"));
```
Comment (chrismaree) [inline packages/optimistic-oracle/test/proposer.js:444]: updated tests accordingly.

**Review by mrice32 — APPROVED** [review]
LGTM!

**Pull request merged**